packages/hint/docs/user-guide/concepts/hints.md

```diff
@@ -82,31 +90,24 @@ using an npm package called `@hint/hint-example1`:
 }
 ```
 
-The `off` and `warning` hint severities may be applied with shorthand
-characters `-` and `?` respectfully when using the array syntax:
+You may use the following characters in place of the associated `severity`.
 
-A hint that has the `off` severity applied:
-
-```json
-{
-    "hints": [
-        "-example1"
-    ]
-}
-```
+| severity | short-hand character |
+|:--- |:--- |
+| `off` | `-` |
+| `warning` | `?` |
 
-A hint that has the `warning` severity applied:
+The following code snippet displays how to configure the hint and set the severity to `off` using the short-hand character.
 
 ```json
 {
     "hints": [
-        "?example1"
+        "-example1"
     ]
 }
 ```
 
-Additionally, some hints allow further customization. The configuration
-in that case it will be similar to the following:
+Some hints enable further customization.  The configuration with further customization should be similar to the following code snippets.
```

Comment:
@molant @antross
The examples are clear, but the scenario should be clarified. When should a user use a customization, or when is the customization available?

Reply:
@molant @antross
Is there an update?